### PR TITLE
misc docs

### PR DESCRIPTION
**Pull request by antoine-levitt**

### CODE REVIEW

File: docs/src/guide/tutorial.jl [hunk 1]
```diff
@@ -50,12 +50,18 @@ scfres = self_consistent_field(basis, tol=1e-8);
...
-# For instance, the energies:
+# For instance, the different components of the energy:
 scfres.energies
-# Eigenvalues: here `scfres.eigenvalues` returns a 7x8 Array where 7 is the
-# number of eigenvalues that are computed and 8 the number of kpoints.
-hcat(scfres.eigenvalues...)
-# Use `scfres.occupation` to see the number of electrons on each energy level.
+
+# Eigenvalues: 
+hcat(scfres.eigenvalues...) # eigenvalues is an array of arrays; this converts it to a matrix
```
Comment (mfherbst) [inline docs/src/guide/tutorial.jl:57]: That's too long a comment, makes it hard to process for new users. Include as explanatory text above.

File: docs/src/index.md [hunk 2]
```diff
@@ -46,6 +46,8 @@ codes such as [Abinit](https://www.abinit.org/),
       DFTK-based calculator implementation for ASE.
     - Read data in [ETSF Nanoquanta](https://doi.org/10.1016/j.commatsci.2008.02.023) format.
 
+Missing a feature? Look for an open issue or create one [here](https://github.com/JuliaMolSim/DFTK.jl/issues).
```
Comment (mfherbst) [inline docs/src/index.md:49]: here is generally bad as a name for a link. What if the link is broken ... I would link `or [create a new one](....).`

File: docs/src/guide/tutorial.jl [hunk 3]
```diff
@@ -54,14 +54,17 @@ scfres = self_consistent_field(basis, tol=1e-8);
...
-hcat(scfres.eigenvalues...) # eigenvalues is an array of arrays; this converts it to a matrix
-
-# This is a 7x8 array: 7 is the number of computed eigenvalues and 8
-# the number of kpoints. There are 7 eigenvalues because there are 4
-# occupied states in the system (4 valence electrons per silicon atom,
-# two atoms per unit cell, and paired spins), and the eigensolver
-# gives itself some breathing room by computing some extra states (see
-# `n_ep_extra` argument to `self_consistent_field`). We can check the occupations:
+hcat(scfres.eigenvalues...)
+# `eigenvalues` is an array (indexed by kpoints) of arrays (indexed by eigenvalue number). The "splatting" operation `...` calls `hcat` with all the inner arrays as arguments, which collects them into a matrix.
```
Comment (mfherbst) [inline docs/src/guide/tutorial.jl:58]: Do you need to make this line so long? :smile:

**Comment by mfherbst** [comment]
Merci :) Now I'm happy.

File: docs/src/guide/tutorial.jl (hunk 3 above)
Comment (antoine-levitt) [reply docs/src/guide/tutorial.jl]: I don't *need* to, it's just fun to make you get mad ;-) https://linux.slashdot.org/story/20/05/31/211211/linus-torvalds-argues-against-80-column-line-length-coding-style-as-linux-kernel-deprecates-it btw

Comment (mfherbst) [reply docs/src/guide/tutorial.jl]: But your line was not 100chars either

Comment (antoine-levitt) [reply docs/src/guide/tutorial.jl]: good point :p

**Pull request merged**